app/backend/src/couchers/config.py

```diff
@@ -23,8 +23,6 @@
     ("ADD_DUMMY_DATA", bool),
     # Email
     ("ENABLE_EMAIL", bool),
-    # Sender name for outgoing notification emails e.g. "Couchers.org"
-    ("NOTIFICATION_EMAIL_SENDER", str),
```

Comment:
Why did you get rid of `NOTIFICATION_EMAIL_SENDER`?

Reply:
Because it was the same string in testing and deploy.